packages/interceptors-opentelemetry/src/workflow/workflow-module-loader.ts

```diff
@@ -30,6 +33,21 @@ export function getWorkflowModule(): typeof WorkflowModule {
   return workflowModule!;
 }
 
+/**
+ * Returns if an SDK flag was set
+ *
+ * Expects to be called only after `ensureWorkflowModuleLoaded`.
+ * Will throw if `@temporalio/workflow` is not available
+ */
+export function hasSdkFlag(flag: keyof SdkFlags): boolean {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  const { SdkFlags } = require('@temporalio/workflow/lib/flags');
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  const { getActivator } = require('@temporalio/workflow/lib/global-attributes');
```

Comment:
Why do we need to import in the function?
Generally - maybe this indicates the function belongs in `@temporalio/workflow/lib` and we just need a single internal import to use this (instead of 3). This file could just re-export it, or import directly in the `index.ts` usage, not sure.

Reply:
Is it the type aliasing that is odd or the fact that we have a top level import and an import in a function?